monkeys/arch-rivals/templates/policy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "chaos-arch-rivals-deny-ingress"
+spec:
+  endpointSelector:
+    matchLabels:
+      name: {{ .Values.name }}
+  ingress:
+  - {}
+  egress:
+  - toEndpoints:
```

Comment:
You need to whitelist IPs to be able to reach the slack hook

Reply:
Added `templates/slack-hook-policy.yaml`.